apps/test/unit/componentLibrary/ToggleTest.jsx

```diff
@@ -0,0 +1,153 @@
+import React from 'react';
+import {render, screen} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import sinon from 'sinon';
+
+import {expect} from '../../util/reconfiguredChai';
+
+import Toggle from '@cdo/apps/componentLibrary/toggle';
+
+describe('Design System - Toggle', () => {
+  it('Toggle - renders with correct label', () => {
+    render(
+      <Toggle name="test-toggle" value="test-toggle" label="Toggle label" />
+    );
+
+    const toggle = screen.getByDisplayValue('test-toggle');
+    expect(toggle).to.exist;
+    expect(screen.getByText('Toggle label')).to.exist;
```

Comment:
nit: my personal preference is to save off label strings like this ('Toggle label') into constants and check against those, just to prevent any unintentional developer error. Not required however.